charm/ksql/lib/charms/layer/ksql.py

```diff
@@ -25,21 +25,30 @@
 from charmhelpers.core.templating import render
 from charmhelpers.core.hookenv import config
 
-KSQL_PORT=8088
 KSQL_SNAP='ksql-server'
 KSQL_SERVICE='snap.{}.ksql-server.service'.format(KSQL_SNAP)
 KSQL_SNAP_COMMON='/var/snap/{}/common'.format(KSQL_SNAP)
-KSQL_CLUSTER_ID='production'
 
 class Ksql(object):
-    def __init__(self):
-        self.cfg = config()
+    def __init__(self, cfg=config()):
+        self.cfg = cfg
+        
+        if self.cfg['use-ssl']:
```

Comment:
See comment above.. if we're not implementing TLS properly, this should be removed.